libs/wire-api/src/Wire/API/Routes/Public/Galley.hs

```diff
@@ -83,501 +83,577 @@ type RemoveFromConversationVerb =
      ]
     (Maybe Event)
 
-data Api routes = Api
-  { -- Conversations
+type ServantAPI =
+  ConversationAPI
+    :<|> TeamConversationAPI
+    :<|> MessagingAPI
+    :<|> TeamAPI
+    :<|> FeatureAPI
```

Comment:
I like the more tree-shaped structure!  Much nicer than the comments over sections of a huge record.